edd_utils/__init__.py

```diff
@@ -137,6 +137,71 @@ def export_study(session, slug, edd_server='edd.jbei.org', verbose=True):
     return df
 
 
+#Get metadata
+def export_metadata(session, slug, edd_server='edd.jbei.org', verbose=True):
+    '''Export a Study from EDD as a pandas dataframe'''
+
+    try:
+        lookup_response = session.get(f'https://{edd_server}/rest/studies/?slug={slug}')
+
+    except KeyError:
```

Comment:
The `get` method of `requests` does not raise a `KeyError`, nor does the string formatting of the URL. You probably want a call to `lookup_response.raise_for_status()` inside the try-block. Check out the [docs for handling response codes](https://2.python-requests.org/en/master/user/quickstart/#response-status-codes).

Reply:
Thats a copy from your original code did not change it

Reply:
Not my code, and it doesn't matter where you copied it from if it's still wrong.